tests/tracking/test_rest_tracking.py

```diff
@@ -99,7 +119,7 @@ def test_create_get_list_experiment(mlflow_client):
     assert exp.artifact_location == 'my_location'
 
     experiments = mlflow_client.list_experiments()
-    assert set([e.name for e in experiments]) == {'My Experiment', 'Default'}
+    assert set([e.name for e in experiments]) == {'My Experiment'}
```

Comment:
The tracking server specified a file store directory that already existed. As a result, a default experiment was not created.